nixos/modules/services/network-filesystems/seaweedfs.nix

```diff
@@ -0,0 +1,597 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.seaweedfs;
+  baseDir = "/var/lib/seaweedfs";
+in
+{
+  options.services.seaweedfs = {
+    package = lib.mkOption {
```

Comment:
Please use mkPackageOption

Also we should have an enable option at this level, not only inside the suboption.